tests/phpunit/tests/post.php

```diff
@@ -842,110 +846,117 @@ public function test_wp_insert_post_handle_malformed_post_date( $input, $expecte
 	}
 
 	/**
-	 * @ticket 26798
+	 * Data provider for test_wp_insert_post_handle_malformed_post_date().
+	 *
+	 * @return array<array{ date: string, expected: bool }>
```

Comment:
Ideally this would be:


```suggestion
	 * @return array<string, array{ date: string, expected: bool }>
```

Where the array keys in the data provider below are strings in an associative array. What are currently comments like "24-hour time format" should rather be part of the array key. This would make it much easier to locate the data set for a failing test.